.config/eslint.mjs

```diff
@@ -0,0 +1,103 @@
+// -----------------------------------------------------------------------------
+// Requirements
+// -----------------------------------------------------------------------------
+
+const configEntries = ['babel.config.js', '.babelrc.js', '.eslintrc.js'];
+const prettierOptions = {
+  arrowParens: 'always',
+  bracketSpacing: true,
+  endOfLine: 'lf',
+  printWidth: 80,
+  quoteProps: 'as-needed',
+  semi: true,
+  singleQuote: true,
+  tabWidth: 2,
+  trailingComma: 'es5',
+  useTabs: false,
+};
+
+const OFF = 0;
+const WARN = 1;
+const ON = 2;
```

Comment:
instead of these, eslint supports the strings "off", "warn", and "error". can we use these inline instead of making constants?

Reply:
We could, but this seems slightly more elegant to me. What is the advantage of using the strings? I can define the constants above the variables if that would be more technically correct. Is the goal to make the file less lines?

Reply:
The advantage is then it looks like an idiomatic eslint config, instead of having constants that are only defined in this one config.

Reply:
A more developed website will eventually have a dynamic Babel config too. One for development, on more production.

Reply:
That's not my experience; you can (and one usually does) configure both in the same static Babel config pivoting on the NODE_ENV env var.

Reply:
Yes, that's what I meant, actually. One config file, but separate configs per environment. I would consider that approach to be pretty dynamic, though. If that's not dynamic, I don't know what is. 😅

Reply:
I guess the outstanding questions I have are whether the use of variables in the root `.eslintrc.js` file makes it _dynamic_, which detracts from its idiomaticness.